qa/tasks/ceph_manager.py

```diff
@@ -1719,25 +1720,10 @@ def do_rados(self, cmd, pool=None, namespace=None, remote=None, **kwargs):
         if remote is None:
             remote = self.controller
 
-        pre = [
-            'adjust-ulimits',
-            'ceph-coverage',
-           f'{self.testdir}/archive/coverage',
-            'rados',
-            '--cluster',
-            self.cluster,
-            ]
-        if pool is not None:
-            pre += ['--pool', pool]
-        if namespace is not None:
-            pre += ['--namespace', namespace]
-        pre.extend(cmd)
-        proc = remote.run(
-            args=pre,
-            wait=True,
-            **kwargs
-            )
-        return proc
+        cmd = (self.rados_cmd + cmd +
```

Comment:
It doesn't make a difference whether we pass option first or subcommand first, right?

Reply:
@batrick

Reply:
It may or may not make a difference but it is confusing. The options are common to all subcommands so, conventionally, they go before the subcommands.

Reply:
Cool